index.html

```diff
@@ -46,7 +46,182 @@ <h1 class="title playfair" >Pritikant Dhal</h1>
   <div>
     <div class="bg"></div>
   </div>
- 
-  
+
+  <!--Project Section-->
+  <section id="projects" class="popins">
+  <h3 id="project-title" class="playfair title">projects</h3>
+  <article class="project flex">
+    <div>
+    <div class="bg1"></div>
+    </div> 
+  <p>Project name goes here</p>
+  <ul class="project-languages flex">
```

Comment:
- [ ] Although you did a commendable😍😍 job by including the project title using the `<p>` tag, it would be more appropriate to use the` <h3>, <h4>, or <h5>` tag as it signifies a title.